include/node_kernels/MomentumActuatorNodeKernel.h

```diff
@@ -0,0 +1,54 @@
+/*------------------------------------------------------------------------*/
+/*  Copyright 2019 National Renewable Energy Laboratory.                  */
```

Comment:
@overfelt You might want to assert Sandia copyright assertion here.

Reply:
Interesting.  Should we have both NREL and SNL in all files for consistency?  Or can only one of us assert copyright at a time?

Reply:
@overfelt We will have to wait for Mike and Paul to weigh in on this one. I think they are working on getting a joint copyright assertion for the codebase.